_rules/aria-state-or-property-valid-value-6a7281.md

```diff
@@ -53,45 +44,51 @@ Some user agents treat the value of `aria-*` attribute as case-sensitive (even w
 - [WAI-ARIA 1.1, Characteristics of States and Properties, Value](https://www.w3.org/TR/wai-aria/#propcharacteristic_value)
 - [Uniform Resource Identifier (URI): Generic Syntax (RFC 3986)](https://www.ietf.org/rfc/rfc3986.txt)
 
+**Note:** Only for [WAI-ARIA required properties][] with value types `ID Reference` and `ID Reference List` is there a requirement that the elements with the given ids actually exists. For non-required properties, this is not a requirement.
+
+**Note:**
+For value type `URI`, this rule does not require that the destination URI exists.
+
 ## Test Cases
 
 ### Passed
 
 #### Passed Example 1
 
-`aria-required` property with valid true/false value
+This `div` element has an `aria-required` property with valid `true/false` value.
 
 ```html
 <div role="textbox" aria-required="true" aria-label="A required textbox"></div>
 ```
 
 #### Passed Example 2
 
-`aria-expanded` state with valid true/false/undefined value
+This `div` element has an `aria-expanded` state with valid `true/false/undefined` value.
 
 ```html
 <div role="button" aria-expanded="undefined">A button</div>
 ```
 
 #### Passed Example 3
 
-`aria-pressed` state with valid tristate value
+This `div` element has an `aria-pressed` state with valid tristate value.
 
 ```html
 <div role="button" aria-pressed="mixed">An other button</div>
 ```
 
 #### Passed Example 4
 
-`aria-errormessage` property with valid ID reference value
+This `div` element has an `aria-errormessage` property with valid ID reference value.
 
 ```html
+<div id="my-error">Please provide a valid username</div>
```

Comment:
As a matter of fact, `aria-errormessage` is **not** required by `textbox`.
I think it is good to show that part of the rule in action. I suggest:
* remove the `div id="my-error"`.
* update the description to say that `aria-errormessage` is not required, so we don't care that the id is missing. Maybe adding a mollifying sentence in the line of "presumably, the error message (with the correct id) will be added by scripting when an error is detected".
* re-order the examples to have "required ID" before "not-required ID" so that a logical sequence follows. (and keep "required ID list" close to them).